packages/next-app/src/app/metadata/(components)/RootLayout.tsx

```diff
@@ -46,6 +48,10 @@ export function RootLayout({ children }: { children: React.ReactNode }) {
       .catch(() => setPoolsData([]));
   }, [chain, address]);
 
+  useEffect(() => {
+    if (chain) push("/metadata");
+  }, [chain]);
```

Comment:
If we are on a pool metadata page and switch the network, the URL does not change, so we redirect to the `/metadata` page 

e.g: The browser is on `http://localhost:3000/metadata/goerli/pool/xxxx` and switches the network to Ethereum. we could redirect to `http://localhost:3000/metadata/mainnet/pool/xxxx` but it doesn't make sense since the pool `xxxx` only exists on the Goerli.  So we redirect the user to the `/metadata` page when the user switches the network. 

Does that make sense?